### PR TITLE
fix: append user specified arguments while launching the mariadb server

### PR DESCRIPTION
**Pull request by sameersbn**
This is a fix for part 1 of #14. Not sure if the image expects users to start mariadb server as specified in part 2 of #14

### CODE REVIEW

**Comment by sameersbn** [comment]
updated to handle both cases described in #14

User specified arguments are respected in both of the following cases. 

```bash
docker run -it --rm bitnami/mariadb --max-connect-errors=1000
```

```bash
docker run -it --rm bitnami/mariadb mysqld.bin --max-connect-errors=1000
```

File: entrypoint.sh [hunk 1]
```diff
@@ -8,6 +8,9 @@ print_welcome_page
 if [ "${1:0:1}" = '-' ]; then
   EXTRA_OPTIONS="$@"
   set -- mysqld.bin
+elif [ "${1}" == "mysqld.bin" -o "${1}" == "/opt/bitnami/mysql/bin/mysqld.bin" ]; then
```
Comment (migmartri) [inline entrypoint.sh]: The path should not be hardcoded.

**Comment by prydonius** [comment]
Thanks @sameersbn! The issue was that EXTRA_OPTIONS is not exported and so we cannot use it in the custom utils (https://github.com/bitnami/bitnami-docker-mariadb/blob/master/bitnami-utils-custom.sh#L2). Do you mind removing the reference to that there as part of this PR?

Thanks for spotting part 2, we will fix it in similar images.

File: entrypoint.sh (hunk 1 above)
Comment (sameersbn) [inline entrypoint.sh]: You are right. My bad.

**Comment by sameersbn** [comment]
@prydonius updated the PR with the suggested changes.

**Comment by prydonius** [comment]
Thank you @sameersbn!

**Pull request merged**